frontend/src/utils.js

```diff
@@ -3,49 +3,46 @@
 // import store from 'store/configureStore'
 
 import { ERROR } from "actions/types";
+import history from "./history";
+import paths from "routes/paths";
 
-const api = async (path, payload = {}) => {
-    // const { auth } = await store.getState();
-    const fullPayload = {
-        method: 'GET',
-        ...payload,
-        headers: {
-            'Content-Type': 'application/json',
-            ...payload.headers,
+const api = async (path, payload = {}, explicitBody = false) => {
+  // const { auth } = await store.getState();
+  const fullPayload = {
+    method: 'GET',
+    ...payload,
+    headers: {
+      'Content-Type': 'application/json',
+      ...payload.headers,
+    },
+    body: explicitBody
+      ? payload.body
+      : JSON.stringify(Object.keys(payload.body || {}).reduce(
+        (acc, key) => {
+          const snakeCaseKey = key.replace( /([A-Z])/g, "_$1").toLowerCase();
+          acc[snakeCaseKey] = payload.body[key];
+          return acc;
         },
-        body: JSON.stringify(Object.keys(payload.body || {}).reduce(
-            (acc, key) => {
-              const snakeCaseKey = key.replace( /([A-Z])/g, "_$1").toLowerCase();
-              acc[snakeCaseKey] = payload.body[key];
-              return acc;
-            },
-            {}
-          ))
-    };
-    // if (auth && auth.token) fullPayload.headers['Authorization'] = `Bearer ${auth.token}`;
+        {}
+      ))
+  };
+  // if (auth && auth.token) fullPayload.headers['WWW-Authenticate'] = `Bearer ${auth.token}`;
```

Comment:
Just double-checking. This is commented because we still haven't implemented any protected endpoint server side, isn't it? So still there is no need to send the bearer token along

Reply:
Precisely, we don't have any secure endpoints yet. However I did change it up from my standard boilerplate-y version to what I know the backend is expecting!